app/serializers/actor_type_taxonomy_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class ActorTypeTaxonomySerializer
+  include FastApplicationSerializer
+
+  attributes :actor_type_id, :taxonomy_id
+
+  set_type :actor_type_taxonomies
```

Comment:
@tmfrnz what should this type be? `actor_type_taxonomies` or plural to both `actor_types_taxonomies` ?